DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -24,6 +24,21 @@ namespace ticl {
     // the first element is on the inner layer and
     // the outer element is on the outer layer.
     std::vector<std::array<unsigned int, 2> > edges;
+
+    // regressed energy
+    float regressed_energy;
+
+    // trackster ID probabilities
+    std::array<float, 4> id_probabilities;
+
+    // convenience methods to return certain id probabilities
+    inline float photon_probability() { return id_probabilities[0]; };
+
+    inline float electron_probability() { return id_probabilities[1]; };
+
+    inline float muon_probability() { return id_probabilities[2]; };
+
+    inline float hadron_probability() { return id_probabilities[3]; };
```

Comment:
is this final?
Will there be a charged_hadron vs neutral_hadron if tracks are added to the game?

Reply:
Good point. I will split this into `charged_hadron_probability()` and `neutral_hadron_probability()` and adjust the current model so that it outputs 5 values with the last one being constant 0 for the moment.

Reply:
to avoid remapping, to the PF particle types, perhaps it would make sense to at least magically align them to the same order (ch, el, mu, ph, nh).

BTW, are these probabilities expected to be uniquely usable only for the Trackster class? I'm thinking if there is a good reason not to wrap them in a struct.